cpp/src/gandiva/gdv_function_stubs_test.cc

```diff
@@ -20,10 +20,19 @@
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
 
+#include <cmath>
+
 #include "gandiva/execution_context.h"
 
 namespace gandiva {
 
+static const double DEFAULT_THRESHOLD = 0.00005;
```

Comment:
Why did you choose this threshold? On fabs documentation it uses 6 places for decimal part, shouldn't it be **0.000005** a better value for AlmostEquals threshold?

Reply:
I think choosing an adequate does not have a simple and unique answer like is shown in these articles:
- https://docs.oracle.com/cd/E19957-01/806-3568/ncg_goldberg.html
- https://randomascii.wordpress.com/2012/02/25/comparing-floating-point-numbers-2012-edition/

I choose that number because it is the value defined in the [float header as default threshold](https://www.cplusplus.com/reference/cfloat/) . For huge and small numbers, I think the developer should set the threshold manually when he is calling the method.

Reply:
I will change the method to use the `DBL_EPSILON` from the macro